ansible_collections/arista/avd/roles/eos_designs/templates/facts/topology/p2p-uplinks.j2

```diff
@@ -33,6 +33,14 @@ topology:
 
       peer_ip_address: {% include switch.ip_addressing.p2p_uplinks_peer_ip %}
 
+{%             endif %}
+{%             if switch.link_tracking_groups is arista.avd.defined %}
```

Comment:
This should also be added to port-channel uplink type

Reply:
will do.